packages/safe-service-client/README.md

```diff
@@ -0,0 +1,233 @@
+### addSafeDelegate
+
+Adds a new delegate for a given Safe address. The signature is calculated by signing this hash: keccak(address + str(int(current_epoch / 3600))).
+
+```js
+await safeService.addSafeDelegate(safeAddress, delegate)
+```
+
+### removeSafeDelegate
+
+Removes a delegate for a given Safe address. The signature is calculated by signing this hash: keccak(address + str(int(current_epoch / 3600))).
+
+```js
+await safeService.removeSafeDelegate(safeAddress, delegate)
+```
+
+### getSafeCreationInfo
+
+Returns the creation information of a Safe.
+
+```js
+const safeCreationInfo: SafeCreationInfoResponse = await safeService.getSafeCreationInfo(safeAddress)
+```
+
+### estimateSafeTransaction
+
+Estimates the safeTxGas for a given Safe multi-signature transaction.
+
+```js
+const estimateTx: SafeMultisigTransactionEstimateResponse = await safeService.estimateSafeTransaction(safeAddress, safeTransaction)
+```
+
+### proposeTransaction
+
+Creates a new multi-signature transaction and stores it in the Safe Transaction Service.
+
+```js
+await safeService.proposeTransaction(safeAddress, transaction, transactionHash, signature)
+```
+
+### getIncomingTransactions
+
+Returns the history of incoming transactions of a Safe account.
+
+```js
+const incomingTxs: TransferListResponse = await safeService.getIncomingTransactions(safeAddress)
+```
+
+### getModuleTransactions
+
+Returns the history of module transactions of a Safe account.
+
+```js
+const moduleTxs: SafeModuleTransactionListResponse = await safeService.getModuleTransactions(safeAddress)
+```
+
+### getMultisigTransactions
+
+Returns the history of multi-signature transactions of a Safe account.
+
+```js
+const multisigTxs: SafeMultisigTransactionListResponse = await safeService.getMultisigTransactions(safeAddress)
+```
+
+### getPendingTransactions
+
+Returns the list of multi-signature transactions that are waiting for the confirmation of the Safe owners.
+
+```js
+const pendingTxs: SafeMultisigTransactionListResponse = await safeService.getPendingTransactions(safeAddress)
+```
+
+```js
+const pendingTxs: SafeMultisigTransactionListResponse = await safeService.getPendingTransactions(safeAddress, currentNonce)
+```
+
+### getBalances
+
+Returns the balances for Ether and ERC20 tokens of a Safe.
+
+```js
+const balances: SafeBalanceResponse[] = await safeService.getBalances(safeAddress)
+```
+
+### getUsdBalances
+
+Returns the balances for Ether and ERC20 tokens of a Safe with USD fiat conversion.
+
+```js
+const usdBalances: SafeBalanceUsdResponse[] = await safeService.getUsdBalances(safeAddress)
+```
+
+### getCollectibles
+
+Returns the collectives (ERC721 tokens) owned by the given Safe and information about them.
+
+```js
+const collectibles: SafeCollectibleResponse[] = await safeService.getCollectibles(safeAddress)
+```
+
+### getTokens
+
+Returns the list of all the ERC20 tokens handled by the Safe.
+
+```js
+const tokens: TokenInfoListResponse = await safeService.getTokens()
```

Comment:
Is there `safeAddress` arg missing? 

EDIT: Ok, it seems to be a token list. Maybe better to rename it to getTokenList and update description